pkg/oc/cli/admin/release/new.go

```diff
@@ -135,6 +135,7 @@ func NewRelease(f kcmdutil.Factory, parentName string, streams genericclioptions
 	flags.BoolVar(&o.ForceManifest, "release-manifest", o.ForceManifest, "If true, a release manifest will be created using --name as the semantic version.")
 
 	// validation
+	flags.BoolVar(&o.AllowCommitDivergence, "allow-commit-divergence", o.AllowCommitDivergence, "If true, allow new releases to reference images built from different commits of a single Git repository.")
```

Comment:
`--allow-inconsistent-sources` probably.  I don't like `divergence` in this context.  `--allow-inconsistent-commits` doesn't make enough sense on its own.

Reply:
> `--allow-inconsistent-sources` probably.

Done with c9d8e9db96 -> 0a1557b5fa.  Updated the commit message and PR topic to match as well.